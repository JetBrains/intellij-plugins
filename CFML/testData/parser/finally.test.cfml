<cfscript>
  try{
  }catch(Any e){
    rethrow;
  }finally{
  }
</cfscript>
