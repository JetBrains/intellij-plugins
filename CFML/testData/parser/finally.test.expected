CfmlFile:finally.cfml
  Tag
    PsiElement(OPENER)('<')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(R_ANGLEBRACKET)('>')
    TRYCATCHEXPRESSION
      PsiElement(TRY_KEYWORD)('try')
      BLOCK_OF_STATEMENTS
        PsiElement(L_CURLYBRACKET)('{')
        PsiElement(R_CURLYBRACKET)('}')
      CATCH_EXPRESSION
        PsiElement(CATCH_KEYWORD)('catch')
        PsiElement(L_BRACKET)('(')
        TYPE
          PsiElement(ANY_TYPE)('Any')
        PsiElement(IDENTIFIER)('e')
        PsiElement(R_BRACKET)(')')
      BLOCK_OF_STATEMENTS
        PsiElement(L_CURLYBRACKET)('{')
        PsiElement(RETHROW_KEYWORD)('rethrow')
        PsiElement(SEMICOLON)(';')
        PsiElement(R_CURLYBRACKET)('}')
      PsiElement(FINALLY_KEYWORD)('finally')
      BLOCK_OF_STATEMENTS
        PsiElement(L_CURLYBRACKET)('{')
        PsiElement(R_CURLYBRACKET)('}')
    PsiElement(LSLASH_ANGLEBRACKET)('</')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(CLOSER)('>')